docs/boot-server/ipxe.cfg.j2

```diff
@@ -0,0 +1,44 @@
+#!ipxe
+
+cpair --foreground 6 --background 9 1
+cpair --foreground 7 --background 9 2
+cpair --foreground 6 --background 9 3
+
+console --picture http://{{ inventory_hostname }}/tateru/ipxe.png
```

Comment:
We should probably not use inventory_hostname here but instead ansible_fqdn or get the actual IP from gather facts.